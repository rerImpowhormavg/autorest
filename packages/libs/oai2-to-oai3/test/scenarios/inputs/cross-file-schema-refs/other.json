{
  "definitions": {
    "MySharedSchema": {
      "type": "object",
      "properties": {
        "name": {
          "type": "string"
        }
      }
    }
  }
}
