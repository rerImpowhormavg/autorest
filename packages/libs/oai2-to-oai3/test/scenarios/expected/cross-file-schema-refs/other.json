{
  "servers": [],
  "components": {
    "schemas": {
      "MySharedSchema": {
        "type": "object",
        "properties": {
          "name": {
            "type": "string"
          }
        }
      }
    }
  }
}